bin/cli.py

```diff
@@ -19,98 +22,46 @@ def train(
     name: str = typer.Option(
         ..., "-n", "--name", help="Name of experiment to save under."
     ),
+    epochs: int = typer.Option(
```

Comment:
nitpick: conventionally long argument names use a double dash, i.e. --epochs. If you want a short flag you can also have a short name, like this:

```
        ..., "-e", "--epochs", help="Number of epochs to run."
```

Reply:
cool, I didn't know that!